Templates/GraphicsGem/Template/Code/Include/${Name}/${Name}TypeIds.h

```diff
@@ -11,16 +11,16 @@
 namespace ${Name}
 {
     // System Component TypeIds
-    inline constexpr const char* ${Name}SystemComponentTypeId = "${SysCompClassId}";
-    inline constexpr const char* ${Name}EditorSystemComponentTypeId = "${EditorSysCompClassId}";
+    inline constexpr const char* ${Name}SystemComponentTypeId = "{${Random_Uuid}}";
+    inline constexpr const char* ${Name}EditorSystemComponentTypeId = "{${Random_Uuid}}";
```

Comment:
These placeholders were fine, it was just the actual Uuids that needed to be replaced with the "{${Random_Uuid}}" placeholder.

Reply:
ok, I'll bring them back